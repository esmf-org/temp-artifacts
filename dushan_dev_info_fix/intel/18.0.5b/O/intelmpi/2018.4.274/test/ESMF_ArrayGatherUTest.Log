2022-07-22 05:59:56
20220722 064207.318 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220722 064207.318 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220722 064207.318 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220722 064207.318 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220722 064207.318 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220722 064207.318 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220722 064207.318 INFO             PET0 Running with ESMF Version   : v8.4.0b07-5-g180888c81e
20220722 064207.318 INFO             PET0 ESMF library build date/time: "Jul 22 2022" "05:57:38"
20220722 064207.318 INFO             PET0 ESMF library build location : /scratch1/NCEPDEV/stmp2/Rocky.Dunlap/esmf-test-dev/intel_18.0.5b_intelmpi_O_dushan_dev_info_fix/esmf
20220722 064207.318 INFO             PET0 ESMF_COMM                   : intelmpi
20220722 064207.319 INFO             PET0 ESMF_MOAB                   : enabled
20220722 064207.319 INFO             PET0 ESMF_LAPACK                 : enabled
20220722 064207.319 INFO             PET0 ESMF_NETCDF                 : enabled
20220722 064207.319 INFO             PET0 ESMF_PNETCDF                : disabled
20220722 064207.319 INFO             PET0 ESMF_PIO                    : enabled
20220722 064207.319 INFO             PET0 ESMF_YAMLCPP                : enabled
20220722 064207.319 INFO             PET0  Beginning Test, file ESMF_ArrayGatherUTest.F90, line 54
20220722 064207.319 INFO             PET0  NUMBER_OF_PROCESSORS           4
20220722 064207.320 INFO             PET0  PASS  ArrayGather 1d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 68
20220722 064207.320 INFO             PET0  PASS  ArrayGather 1d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, line 77
20220722 064207.320 INFO             PET0  PASS  ArrayGather 1d test, contiguous Array, deBlockList DistGrid, ESMF_ArrayGatherUTest.F90, line 86
20220722 064207.320 INFO             PET0  PASS  ArrayGather 1d test, non-contiguous Array, deBlockList DistGrid, ESMF_ArrayGatherUTest.F90, line 95
20220722 064207.320 INFO             PET0  PASS  ArrayGather 2d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 103
20220722 064207.320 INFO             PET0  PASS  ArrayGather 2d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, line 111
20220722 064207.320 INFO             PET0  PASS  ArrayGather 3d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 119
20220722 064207.328 INFO             PET0  PASS  ArrayGather 3d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, line 127
20220722 064207.328 INFO             PET0  Ending Test, file ESMF_ArrayGatherUTest.F90, line 129
20220722 064207.328 INFO             PET0 Finalizing ESMF
20220722 064207.318 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220722 064207.318 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220722 064207.318 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220722 064207.318 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220722 064207.318 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220722 064207.318 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220722 064207.318 INFO             PET1 Running with ESMF Version   : v8.4.0b07-5-g180888c81e
20220722 064207.318 INFO             PET1 ESMF library build date/time: "Jul 22 2022" "05:57:38"
20220722 064207.318 INFO             PET1 ESMF library build location : /scratch1/NCEPDEV/stmp2/Rocky.Dunlap/esmf-test-dev/intel_18.0.5b_intelmpi_O_dushan_dev_info_fix/esmf
20220722 064207.318 INFO             PET1 ESMF_COMM                   : intelmpi
20220722 064207.319 INFO             PET1 ESMF_MOAB                   : enabled
20220722 064207.319 INFO             PET1 ESMF_LAPACK                 : enabled
20220722 064207.319 INFO             PET1 ESMF_NETCDF                 : enabled
20220722 064207.319 INFO             PET1 ESMF_PNETCDF                : disabled
20220722 064207.319 INFO             PET1 ESMF_PIO                    : enabled
20220722 064207.319 INFO             PET1 ESMF_YAMLCPP                : enabled
20220722 064207.319 INFO             PET1  Beginning Test, file ESMF_ArrayGatherUTest.F90, line 54
20220722 064207.319 INFO             PET1  NUMBER_OF_PROCESSORS           4
20220722 064207.320 INFO             PET1  PASS  ArrayGather 1d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 68
20220722 064207.320 INFO             PET1  PASS  ArrayGather 1d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, line 77
20220722 064207.320 INFO             PET1  PASS  ArrayGather 1d test, contiguous Array, deBlockList DistGrid, ESMF_ArrayGatherUTest.F90, line 86
20220722 064207.320 INFO             PET1  PASS  ArrayGather 1d test, non-contiguous Array, deBlockList DistGrid, ESMF_ArrayGatherUTest.F90, line 95
20220722 064207.320 INFO             PET1  PASS  ArrayGather 2d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 103
20220722 064207.320 INFO             PET1  PASS  ArrayGather 2d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, line 111
20220722 064207.320 INFO             PET1  PASS  ArrayGather 3d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 119
20220722 064207.328 INFO             PET1  PASS  ArrayGather 3d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, line 127
20220722 064207.328 INFO             PET1  Ending Test, file ESMF_ArrayGatherUTest.F90, line 129
20220722 064207.328 INFO             PET1 Finalizing ESMF
20220722 064207.318 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220722 064207.318 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220722 064207.318 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220722 064207.318 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220722 064207.318 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220722 064207.318 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220722 064207.318 INFO             PET2 Running with ESMF Version   : v8.4.0b07-5-g180888c81e
20220722 064207.318 INFO             PET2 ESMF library build date/time: "Jul 22 2022" "05:57:38"
20220722 064207.318 INFO             PET2 ESMF library build location : /scratch1/NCEPDEV/stmp2/Rocky.Dunlap/esmf-test-dev/intel_18.0.5b_intelmpi_O_dushan_dev_info_fix/esmf
20220722 064207.318 INFO             PET2 ESMF_COMM                   : intelmpi
20220722 064207.319 INFO             PET2 ESMF_MOAB                   : enabled
20220722 064207.319 INFO             PET2 ESMF_LAPACK                 : enabled
20220722 064207.319 INFO             PET2 ESMF_NETCDF                 : enabled
20220722 064207.319 INFO             PET2 ESMF_PNETCDF                : disabled
20220722 064207.319 INFO             PET2 ESMF_PIO                    : enabled
20220722 064207.319 INFO             PET2 ESMF_YAMLCPP                : enabled
20220722 064207.319 INFO             PET2  Beginning Test, file ESMF_ArrayGatherUTest.F90, line 54
20220722 064207.319 INFO             PET2  NUMBER_OF_PROCESSORS           4
20220722 064207.320 INFO             PET2  PASS  ArrayGather 1d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 68
20220722 064207.320 INFO             PET2  PASS  ArrayGather 1d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, line 77
20220722 064207.320 INFO             PET2  PASS  ArrayGather 1d test, contiguous Array, deBlockList DistGrid, ESMF_ArrayGatherUTest.F90, line 86
20220722 064207.320 INFO             PET2  PASS  ArrayGather 1d test, non-contiguous Array, deBlockList DistGrid, ESMF_ArrayGatherUTest.F90, line 95
20220722 064207.320 INFO             PET2  PASS  ArrayGather 2d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 103
20220722 064207.320 INFO             PET2  PASS  ArrayGather 2d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, line 111
20220722 064207.320 INFO             PET2  PASS  ArrayGather 3d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 119
20220722 064207.328 INFO             PET2  PASS  ArrayGather 3d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, line 127
20220722 064207.328 INFO             PET2  Ending Test, file ESMF_ArrayGatherUTest.F90, line 129
20220722 064207.328 INFO             PET2 Finalizing ESMF
20220722 064207.318 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220722 064207.318 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220722 064207.318 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220722 064207.318 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220722 064207.318 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220722 064207.318 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220722 064207.318 INFO             PET3 Running with ESMF Version   : v8.4.0b07-5-g180888c81e
20220722 064207.318 INFO             PET3 ESMF library build date/time: "Jul 22 2022" "05:57:38"
20220722 064207.318 INFO             PET3 ESMF library build location : /scratch1/NCEPDEV/stmp2/Rocky.Dunlap/esmf-test-dev/intel_18.0.5b_intelmpi_O_dushan_dev_info_fix/esmf
20220722 064207.318 INFO             PET3 ESMF_COMM                   : intelmpi
20220722 064207.319 INFO             PET3 ESMF_MOAB                   : enabled
20220722 064207.319 INFO             PET3 ESMF_LAPACK                 : enabled
20220722 064207.319 INFO             PET3 ESMF_NETCDF                 : enabled
20220722 064207.319 INFO             PET3 ESMF_PNETCDF                : disabled
20220722 064207.319 INFO             PET3 ESMF_PIO                    : enabled
20220722 064207.319 INFO             PET3 ESMF_YAMLCPP                : enabled
20220722 064207.319 INFO             PET3  Beginning Test, file ESMF_ArrayGatherUTest.F90, line 54
20220722 064207.319 INFO             PET3  NUMBER_OF_PROCESSORS           4
20220722 064207.320 INFO             PET3  PASS  ArrayGather 1d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 68
20220722 064207.320 INFO             PET3  PASS  ArrayGather 1d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, line 77
20220722 064207.320 INFO             PET3  PASS  ArrayGather 1d test, contiguous Array, deBlockList DistGrid, ESMF_ArrayGatherUTest.F90, line 86
20220722 064207.320 INFO             PET3  PASS  ArrayGather 1d test, non-contiguous Array, deBlockList DistGrid, ESMF_ArrayGatherUTest.F90, line 95
20220722 064207.320 INFO             PET3  PASS  ArrayGather 2d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 103
20220722 064207.320 INFO             PET3  PASS  ArrayGather 2d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, line 111
20220722 064207.320 INFO             PET3  PASS  ArrayGather 3d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 119
20220722 064207.328 INFO             PET3  PASS  ArrayGather 3d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, line 127
20220722 064207.328 INFO             PET3  Ending Test, file ESMF_ArrayGatherUTest.F90, line 129
20220722 064207.328 INFO             PET3 Finalizing ESMF
