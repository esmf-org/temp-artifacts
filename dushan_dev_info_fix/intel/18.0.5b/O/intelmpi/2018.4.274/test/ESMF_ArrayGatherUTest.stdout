2022-07-22 05:59:56
  Beginning Test, file ESMF_ArrayGatherUTest.F90, line 54
  Beginning Test, file ESMF_ArrayGatherUTest.F90, line 54
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_ArrayGatherUTest.F90, line 54
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_ArrayGatherUTest.F90, line 54
  NUMBER_OF_PROCESSORS           4
  PASS  ArrayGather 1d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 6
 8
  PASS  ArrayGather 1d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 6
 8
  PASS  ArrayGather 1d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 6
 8
  PASS  ArrayGather 1d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 6
 8
  PASS  ArrayGather 1d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, li
 ne 77
  PASS  ArrayGather 1d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, li
 ne 77
  PASS  ArrayGather 1d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, li
 ne 77
  PASS  ArrayGather 1d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, li
 ne 77
  PASS  ArrayGather 1d test, contiguous Array, deBlockList DistGrid, ESMF_ArrayG
 atherUTest.F90, line 86
  PASS  ArrayGather 1d test, contiguous Array, deBlockList DistGrid, ESMF_ArrayG
 atherUTest.F90, line 86
  PASS  ArrayGather 1d test, contiguous Array, deBlockList DistGrid, ESMF_ArrayG
 atherUTest.F90, line 86
  PASS  ArrayGather 1d test, contiguous Array, deBlockList DistGrid, ESMF_ArrayG
 atherUTest.F90, line 86
  PASS  ArrayGather 1d test, non-contiguous Array, deBlockList DistGrid, ESMF_Ar
 rayGatherUTest.F90, line 95
  PASS  ArrayGather 1d test, non-contiguous Array, deBlockList DistGrid, ESMF_Ar
 rayGatherUTest.F90, line 95
  PASS  ArrayGather 1d test, non-contiguous Array, deBlockList DistGrid, ESMF_Ar
 rayGatherUTest.F90, line 95
  PASS  ArrayGather 1d test, non-contiguous Array, deBlockList DistGrid, ESMF_Ar
 rayGatherUTest.F90, line 95
  PASS  ArrayGather 2d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 1
 03
  PASS  ArrayGather 2d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 1
 03
  PASS  ArrayGather 2d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 1
 03
  PASS  ArrayGather 2d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 1
 03
  PASS  ArrayGather 2d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, li
 ne 111
  PASS  ArrayGather 2d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, li
 ne 111
  PASS  ArrayGather 2d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, li
 ne 111
  PASS  ArrayGather 2d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, li
 ne 111
  PASS  ArrayGather 3d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 1
 19
  PASS  ArrayGather 3d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 1
 19
  PASS  ArrayGather 3d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 1
 19
  PASS  ArrayGather 3d test, contiguous Array, ESMF_ArrayGatherUTest.F90, line 1
 19
  PASS  ArrayGather 3d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, li
 ne 127
  PASS  ArrayGather 3d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, li
 ne 127
  Ending Test, file ESMF_ArrayGatherUTest.F90, line 129
  PASS  ArrayGather 3d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, li
 ne 127
  PASS  ArrayGather 3d test, non-contiguous Array, ESMF_ArrayGatherUTest.F90, li
 ne 127
  Ending Test, file ESMF_ArrayGatherUTest.F90, line 129
  Ending Test, file ESMF_ArrayGatherUTest.F90, line 129
  Ending Test, file ESMF_ArrayGatherUTest.F90, line 129
  PET           0  Test Elapsed Time    9.75000000000000       msec.
  PET           3  Test Elapsed Time    9.81700000000000       msec.
  PET           2  Test Elapsed Time    9.82800000000000       msec.
  PET           1  Test Elapsed Time    10.1470000000000       msec.
